shell/models/provisioning.cattle.io.cluster.js

```diff
@@ -134,6 +134,10 @@ export default class ProvCluster extends SteveModel {
     this.currentRouter().push(location);
   }
 
+  get canDelete() {
+    return this.stateObj.transitioning === true && this.stateObj.name !== 'active' && this._canDelete;
```

Comment:
I don't understand this logic - its doesn't appear to work - you can no longer delete a cluster when it is in the active state. The extra logic here for `this.stateObj.name !== 'active'` does not do anything, sine active if not a transitioning state.

I think you can just use:

```
  get canDelete() {
    return super.canDelete && this.stateObj.name !== 'removing';
  }
```

Note use should use `super.canDelete` rather than `this._canDelete`